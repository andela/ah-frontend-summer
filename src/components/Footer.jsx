import React from 'react';
import { Menu, Container, Icon, List, Segment } from "semantic-ui-react";
import '../styles/footer.css';

const Footer = () => (
    <div>
        <Menu borderless className="footerMenu">
            <Segment vertical>
                <Container textAlign="center">
                    <Icon size="large" name="facebook f" />
                    <Icon size="large" name="instagram" />
                    <Icon size="large" name="twitter" />
                </Container>
                <Container textAlign="center">
                    <List horizontal size="small">
                        <List.Item>
                            <List.Content>
                            Â© 2019 by Author&apos;s Haven. Proudly created with Andela
                            </List.Content>
                        </List.Item>
                    </List>
                </Container>
            </Segment>
        </Menu>
    </div>
);

export default Footer;
