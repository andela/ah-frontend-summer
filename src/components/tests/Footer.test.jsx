import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';

describe('Footer test', () => {
    const wrapper = shallow(<Footer />);
    test('render component without failing', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
