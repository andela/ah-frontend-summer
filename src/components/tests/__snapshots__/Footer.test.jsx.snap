// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test render component without failing 1`] = `
<div>
  <Menu
    borderless={true}
    className="footerMenu"
    fixed="bottom"
  >
    <Segment
      vertical={true}
    >
      <Container
        textAlign="center"
      >
        <Icon
          as="i"
          name="facebook f"
          size="large"
        />
        <Icon
          as="i"
          name="instagram"
          size="large"
        />
        <Icon
          as="i"
          name="twitter"
          size="large"
        />
      </Container>
      <Container
        textAlign="center"
      >
        <List
          horizontal={true}
          size="small"
        >
          <ListItem>
            <ListContent>
              Â© 2019 by Author's Haven. Proudly created with Andela
            </ListContent>
          </ListItem>
        </List>
      </Container>
    </Segment>
  </Menu>
</div>
`;
